fix: changed the code snippet for setting compiler option - KT-60969 (#3944)
JetBrains/kotlin-web-site: docs/topics/js/js-modules.md

- [Asynchronous Module Definitions (AMD)](https://github.com/amdjs/amdjs-api/wiki/AMD), which is in particular
   used by the [RequireJS](https://requirejs.org/) library.
- [CommonJS](http://wiki.commonjs.org/wiki/Modules/1.1), widely used by Node.js/npm
   (`require` function and `module.exports` object).
- Plain. Don't compile for any module system. You can access a module by its name in the global scope.

## Browser targets
 

> If you're targeting the browser and want to use a different module system than UMD, you can specify the desired module

Reviewer: ```suggestion
If you're targeting the browser and want to use a module system other than UMD, you can specify the desired module
```

Reviewer: It might be even better to restructure this a little to something like:
"If you intend to run your code in a web browser environment, and want to use a module system..."

> Webpack provides two different "flavors" of CommonJS, `commonjs` and `commonjs2`, which affect the way your declarations
> are made available. While in most cases, you probably want `commonjs2`, which adds the `module.exports` syntax to the
> generated library, you can also opt for the "pure" `commonjs` option, which implements the CommonJS specification exactly.

Reviewer: ```suggestion
Webpack provides two different "flavors" of CommonJS, `commonjs` and `commonjs2`, which affect the way your declarations
are made available. In most cases, you probably want `commonjs2`, which adds the `module.exports` syntax to the
generated library. Alternatively, you can also opt for the "pure" `commonjs` option, which adheres strictly to the CommonJS specification.
```

To learn more about the difference between `commonjs` and `commonjs2`, see the [Webpack repository](https://github.com/webpack/webpack/issues/1114).

## JavaScript libraries and Node.js files

> If you are creating a library to be consumed from JavaScript or a Node.js file, and want to use a different module

Reviewer: ```suggestion
If you are creating a library for use in JavaScript or a Node.js environments, and want to use a different module
```

system, the instructions are slightly different.

### Choose the target module system

> To select module kind, set the `moduleKind` compiler option in the Gradle build script:

Reviewer: I would suggest changing it to: "To select the target module system" to be consistent with the title and the previous description.

Kotlin is distributed with the Kotlin/JS standard library as a single file, which is itself compiled as an UMD module,

> so you can use it with any module system described above. While for most use cases of Kotlin/JS, it is recommended to use

Reviewer: ```suggestion
so you can use it with any module system described above. For most use cases of Kotlin/JS, it is recommended to use
```
